website/content/en/status/report-2023-01-2023-03/cloud-init.adoc

```diff
@@ -0,0 +1,27 @@
+=== FreeBSD as a Tier 1 cloud-init Platform
+
+Links: +
+link:https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[https://cloud-init.io/] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[https://cloudinit.readthedocs.io/en/latest/] +
+link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[cloud-init ongoing refactorization] URL: link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst]
+
+Contact: Mina Galić <FreeBSD@igalic.co>
+
+cloud-init is the standard way of provisioning servers in the cloud.
+Unfortunately, cloud-init support for operating systems other than Linux is rather poor, and the lack of cloud-init support on FreeBSD is a hindrance to cloud providers who want to offer FreeBSD as a Tier 1 platform.
+To remedy the situation, this project aims to bring FreeBSD cloud-init support on par with Linux support.
+The broader plan is to lift support across all BSDs.
+
+This quarter has been going very, very slowly, for personal reasons — but also for lack of access to right resources.
+I have been trying to link:https://github.com/canonical/cloud-init/pull/2003[port the Infiniband functions].
+This has proven difficult, because it falsified my thesis that man:ifconfig[8] is all that's needed to figure out network interfaces on FreeBSD.
+
+A cloud-init colleague who works for Azure managed to give me access to an link:https://learn.microsoft.com/en-us/azure/architecture/topics/high-performance-computing[HPC VM on Azure].
+Unfortunately, it was only for a limited time, and that was not enough to figure out how to get Infiniband up and running on FreeBSD — a task handled by package:sysutils/azure-agent[] by Linux, but Azure Agent's FreeBSD is rather lacking.
+
+People interested in helping with this project could provide man:ifconfig[8], man:ibstat[8], man:ibv_devinfo[8], etc… link:https://lists.freebsd.org/archives/freebsd-infiniband/2023-February/000005.html[pastes from their Infiniband systems].
+I would also be very happy about getting to access to hardware with Infiniband NICs, or hearing from people who have successfully used FreeBSD on Azure HPC with Infiniband.
```

Comment:
`getting to access to hardware` sounds wrong to me: shouldn't it be `getting access to hardware`?